targets/python/templates/PlayFabUtil.py.ejs

```diff
@@ -0,0 +1,59 @@
+# using PlayFab.Json;
+import json
+<% if (hasClientOptions) {
+%>import PlayFabClientModels
+<% } else if (hasServerOptions) {
+%>import PlayFabServerModels
+<% } %>
+
+class PlayFab:
+    [AttributeUsage(AttributeTargets.Field, AllowMultiple = false)]
+    class Unordered(attribute):
+        def __init__():
+            pass
+
+        def __init__(self, sortProperty):
```

Comment:
I don't think there's overloading in Python?

Reply:
There is.

Reply:
Do you have docs about it? As far as I knew you overload in Python by adding optional arguments (eg. def __init__(self, sortProperty=None))